aten/src/ATen/native/nested/NestedTensorUtils.h

```diff
@@ -315,7 +316,7 @@ inline Tensor wrap_tensor_node(
       TensorOptions().dtype(dtype).layout(layout).device(device).pinned_memory(
           pin_memory);
   if (tensor_node.degree() == 0) {
-    return wrap_buffer(ones({0}, dtype, layout, device), ones({}));
+    return wrap_buffer(ones({0}, at::kLong, at::kStrided, device), ones({}));
```

Comment:
This seems like you're just ignoring dtype and layout entirely. What's going on?

Reply:
was trying to fix the fix the size returning float, but looks like I updated the wrong arg lol